src/mapper/__tests__/mapAnyToOrderFilter.test.ts

```diff
@@ -0,0 +1,64 @@
+import { mapAnyToOrderFilter } from '../mapAnyToOrderFilter';
+import { SortField, SortOrder } from '@airswap/types';
+
+describe("mapAnyToRequestFilter", () => {
+    test("should map all string values", () => {
+        expect(mapAnyToOrderFilter({
+            senderMaxAmount: "20",
+            signerMaxAmount: "200",
+            senderMinAmount: "2",
+            signerMinAmount: "200",
+            senderIds: ["ETH"],
+            signerIds: ["WETH"],
+            limit: 10,
+            offset: 30,
+            senderTokens: ["DAI", "btc"],
+            signerTokens: ["ETH", "ast"],
+            sortField: "SENDER_AMOUNT",
+            sortOrder: "DESC",
+        }, 10)).toEqual({
+            senderMaxAmount: BigInt(20),
+            signerMaxAmount: BigInt(200),
+            senderMinAmount: BigInt(2),
+            signerMinAmount: BigInt(200),
+            limit: 10,
+            offset: 30,
+            senderTokens: ["dai", "btc"],
+            signerTokens: ["eth", "ast"],
+            senderIds: ["eth"],
+            signerIds: ["weth"],
+            sortField: SortField.SENDER_AMOUNT,
+            sortOrder: SortOrder.DESC,
+        });
+    });
+
+    test("Restrain limit to configured", () => {
+        expect(mapAnyToOrderFilter({
+            senderMaxAmount: "20",
+            signerMaxAmount: "200",
+            senderMinAmount: "2",
+            signerMinAmount: "200",
+            senderIds: ["ETH"],
+            signerIds: ["WETH"],
```

Comment:
This should be a number, it's the id for the nft.

```suggestion
            senderIds: [1],
            signerIds: [2],
```